resources/aws_ec2.go

```diff
@@ -58,6 +58,18 @@ const (
 	waitTimeout = 400
 )
 
+//go:generate stringer -type=msgContents -output=msgcontents_stringer.go
+
+// msgContents represents the contents of messages sent via awsChan.
+type msgContents uint8
+
+const (
+	msgContentsRunMgmt         msgContents = iota // 0
```

Comment:
remove the // 0 comments please

also please rename from `RunMgmt` to something else that doesn't have `mgmt` string in it. Maybe `ServerRunning` ??

Could we also name the type `awsEc2Event` or something similar please?